facebook.js

```diff
@@ -5,25 +5,24 @@ const locale = (Array.from(document.body.classList).find(cls => cls.match(/^Loca
  * @param localeData
  */
 const show_facebook_cv_tags = function(localeData) {
-  const images = [...document.getElementsByTagName('img')];
-  const localeRegex = new RegExp(localeData.separator_regex, 'i');
+  const images = [...document.querySelectorAll(`img[alt^="${localeData.tag_prefix}"]`)];
```

Comment:
Using `querySelectorAll` is much slower than using `getElementsByTagName`. Since this event fires every time the page mutates, it wouldn't be wise to make this function slower. It might impact the user experience negatively.

Check out http://stackoverflow.com/a/39332340